README.md

```diff
@@ -120,6 +121,7 @@ while True:
 * Lolin D32 Pro (ESP32 with 4MB PSRAM)
 * Adafruit Feather Huzzah ESP8266
 * Adafruit Feather Huzzah ESP32
+* Raspberry Pico
```

Comment:
`Raspberry Pi Pico`

Reply:
Agreed; I will update the PR code accordingly.